### ISSUE TITLE
.asc import of public file from mail

### ISSUE THREAD

**Issue by flexspr** [lucanaef/PGPro#64]
Hi, sorry maybe I am too dumb, but why can I not import an .asc public key from apple mail into PGPro. I really like the app, but most of the keys get shared with me this way. If I cannot import them, then the use case gets quite difficult. When I click on an .asc public key in apple mail and choose PGPro, then PGPro tells me 0 keys imported, 0 unsupported keys skipped, 0 duplicate keys skipped. Then I click ok, thats it. Am I doing something wrong? The public that I used was created by Enigmail client in Mozilla Thunderbird, Ubuntu GnuPG Key.

**Comment by lucanaef** [comment]
You are right, this should work. Does this also happen in the newest version (2021.4)? If so, please send a .asc public key to [dev@pgpro.app](mailto:dev@pgpro.app) so I can replicate the issue.
